stdlib/@tests/stubtest_allowlists/linux.txt

```diff
@@ -35,6 +34,10 @@ multiprocessing.popen_spawn_win32
 fcntl.I_[A-Z0-9_]+
 os.SCHED_[A-Z_]+
 posix.SCHED_[A-Z_]+
+((os|posix).CLONE_[A-Z]+)?  # Python 3.12+
```

Comment:
While the CLONE_ constants were only added in Python 3.12, I think having them in the common file is better for maintainability.